src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionService.cs

```diff
@@ -40,6 +41,7 @@ internal AbstractTypeImportCompletionService(Workspace workspace)
             bool forceCacheCreation,
             CancellationToken cancellationToken)
         {
+            var hideAdvancedmembers = currentProject.Solution.Workspace.Options.GetOption(CompletionOptions.HideAdvancedMembers, currentProject.Language);
```

Comment:
```suggestion
            var hideAdvancedmembers = currentProject.Solution.Options.GetOption(CompletionOptions.HideAdvancedMembers, currentProject.Language);
```

(fix elsewhere if you used teh former).

the former is mutable and may change as the feature is running.  the latter is an immutable snapshot.

Reply:
Good point. I guess this one needs to be fixed too
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/Shared/Extensions/DocumentExtensions.cs,26